src/main/resources/locales/en-gb.yml

```diff
@@ -25,9 +22,18 @@ edit_name_title: 'Edit Name of %1%'
 edit_description_title: 'Edit Description of %1%'
 edit_privacy_button: '[Edit Privacy](#00fb9a)'
 delete_button: '[Delete](#00fb9a)'
+delete_button_describe: '[  • Right Click: Delete](gray)'
 item_info_name: '[Name:](#00fb9a) %1%'
 item_info_description: '%1%'
 item_info_world: '[World:](#00fb9a) %1%'
 item_info_server: '[Server:](#00fb9a) %1%'
 item_info_coordinates: '[Coordinates:](#00fb9a) x: %1%, y: %2%, z: %3%'
-updated_icon: '[Successfully updated the icon for](#00fb9a) [%1%](#00fb9a bold)'
\ No newline at end of file
+item_controls: '[  • Left Click: Teleport](gray)\n[  • Right Click: Edit](gray)\n[  • Place Item: Set icon](gray)\n'
+item_deleted_name: '&c[DEL] [%1%](#00fb9a)'
+edit_location_default_message: '[Coordinates:](#00fb9a) x: %1%, y: %2%, z: %3%'
+edit_description_default_message: '&7ℹ %1%'
+edit_description_default_message_blank: ''
+edit_description_default_input: 'null' # Leave blank '' to fill in the icon item name, enter a blank ' ' to empty the description
```

Comment:
I dislike having 'null' as a string be the default here -- can it be blank by default, or specify an option to not use it rather than having the user modify the locales?

Reply:
If left blank, it will display the item's name by default.